src/pages/projects/ProjectPage.tsx

```diff
@@ -1,170 +1,150 @@
+  ITokenPaginationData,
+  tokenPaginationInit,
+} from "../../container/tokenPagination/TokenPagination";
+import JSONContainer from "../../components/JSONContainer/JSONContainer";
+import { useDialogContext } from "../../container/app/DialogContext";
+import { PROJECT_DATA_SAMPLE } from "../../utils/Constant";
+import CustomTable, { IColumnMapping } from "../../container/table/Table";
+
+const COLUMN_MAPPPING: IColumnMapping[] = [
+  { displayName: "Name", jsonKeys: ["name"] },
+  { displayName: "Data", jsonKeys: "DATA" },
+  { displayName: "ID", jsonKeys: ["id"] },
+  { displayName: "Owner Id", jsonKeys: ["ownerId"] },
+  { displayName: "Tenant Id", jsonKeys: ["tenantId"] },
+  { displayName: "Tenant Name", jsonKeys: ["tenantName"] },
+  { displayName: "Active", jsonKeys: ["active"] },
+  { displayName: "Information", jsonKeys: ["information"] },
+  { displayName: "Time Created", jsonKeys: ["timeCreated"] },
+  { displayName: "Time Modified", jsonKeys: ["timeModified"] },
+  { displayName: "Billing Mode", jsonKeys: ["billingMode"] },
+  { displayName: "DataSharing Enabled", jsonKeys: ["dataSharingEnabled"] },
+  { displayName: "StorageBundle Id", jsonKeys: ["storageBundle", "id"] },
   {
-    displayName: "StorageBundleName",
-    jsonKey: ["storageBundle", "bundleName"],
+    displayName: "Storage Bundle Name",
+    jsonKeys: ["storageBundle", "bundleName"],
   },
 ];
 
-function CustomTableHead() {
-  return (
-    <TableHead>
-      <TableRow>
-        {COLUMN_CONSTANT.map((columnObject: ITableConstant, index: number) => (
-          <StyledTableCell key={index}>
-            {columnObject.displayName}
-          </StyledTableCell>
-        ))}
-      </TableRow>
-    </TableHead>
-  );
-}
-
-interface IPropsCustomTableBody {
-  listItem: any[];
-}
-
-function findValueFromKeyList(resObj: any, keyList: string[]): string {
-  let name_of_object = resObj;
-  for (const key of keyList) {
-    name_of_object = name_of_object[key];
-  }
-  return String(name_of_object);
-}
-
-function CustomTableBody(props: IPropsCustomTableBody) {
-  const { listItem } = props;
-
-  return (
-    <TableBody>
-      {listItem.map((item: any, index: number) => (
-        <StyledTableRow key={index}>
-          {COLUMN_CONSTANT.map(
-            (displayObj: ITableConstant, objIndex: number) => (
-              <StyledTableCell align="right" key={objIndex}>
-                {typeof displayObj.jsonKey == "string"
-                  ? displayObj.jsonKey
-                  : findValueFromKeyList(item, displayObj.jsonKey)}
-              </StyledTableCell>
-            )
-          )}
-        </StyledTableRow>
-      ))}
-    </TableBody>
-  );
-}
-
-async function getProjectData(): Promise<ProjectPagedList> {
+async function getProjectData(parameter: any): Promise<ProjectPagedList> {
   // Generate axios parameter
   const ProjectParamCreator = ProjectApiAxiosParamCreator();
   const getProjectsParamter = await ProjectParamCreator.getProjects();
```

Comment:
Small typo, probably should change to `getProjectsParameter` or `getProjectsParam` (here and below).